relay-server/Cargo.toml

```diff
@@ -70,8 +70,8 @@ serde = { version = "1.0.114", features = ["derive"] }
 serde_json = "1.0.55"
 serde_urlencoded = "0.7.0"
 smallvec = { version = "1.4.0", features = ["serde"] }
-symbolic-common = { version = "8.7.2", optional = true, default-features=false }
-symbolic-unreal = { version = "8.7.2", optional = true, default-features=false, features=["serde"] }
+symbolic-common = { version = "10.1.2", optional = true, default-features=false }
+symbolic-unreal = { version = "10.1.2", optional = true, default-features=false, features=["serde"] }
```

Comment:
Symbolic 9 contains a list of breaking changes, do any of those apply to us? https://github.com/getsentry/symbolic/blob/master/CHANGELOG.md#900